### ISSUE TITLE
Opening D:/*.log leads to a crash

### ISSUE THREAD

**Issue by barisdemiray** [notepad-plus-plus/notepad-plus-plus#4946]
### Description of the Issue
I wanted to copy and paste a path given by a Visual Studio command, to open all the `.log` files and it led to a crash. Path I entered was this, `D:/*.log`.

It happens with version 7.5.8 (32-bit). With French interface if that matters.

### Steps to Reproduce the Issue
1. Ctrl+O or File/Open.
2. Paste `D:/*.log` in the file name input box.
3. Click Open.

### Expected Behavior
Opening all the `*.log` files in `D:\`.

### Actual Behavior
Crash.

### Debug Information
I saved a crash dump, [NppDump.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/2501000/NppDump.zip).

PS: I got a `Cannot save crash dump (error: 5)` message while trying to save a crash dump. A more clear message like `Cannot save a crash dump under Program Files without Administrator rights` would make more people to send a bug report.

**Comment by SinghRajenM** [comment]
Don't you think `D:/*.log` should be `D:\*.log` (notice slash). Moreover, npp does not supports wildcard for file opening.

**Comment by SinghRajenM** [comment]
@jsnells1 really?
If I am not wrong, it will just open the folder path, then manually you need to select the file(s) and click ok or again you have type *.* or some *.log or similar.

for example <kbd>Ctrl</kbd>+ <kbd>O</kbd> then type `C:\Users\Xyz\Desktop\Garbage\*.txt` will open path till `C:\Users\Xyz\Desktop\Garbage`, then again either you have to select all files or type `*.txt` again, right?

**Comment by jsnells1** [comment]
Oh, sorry you're right. It navigates to the directory and then you can do *.txt once there. I thought you meant all wildcard file opening wasn't supported so I chimed in.

**Comment by sasumner** [comment]
This is fraught with difficulty, and most of it comes from the setting:  "Use new style dialog (without file extension feature and Unix style path capacity"...

**Comment by barisdemiray** [comment]
> Don't you think `D:/*.log` should be `D:\*.log` (notice slash). Moreover, npp does not supports wildcard for file opening.

I agree but we have `Cmake` as the build system and we do slash replacement to avoid escaping. That path probably must be one of them. In any case, end result should not be a crash.

**Comment by sasumner** [comment]
And shouldn't `D:/*.log` actually do something useful when "Unix style path capacity" is enabled?

**Comment by SinghRajenM** [comment]
> In any case, end result should not be a crash.

Yes, I agree with you. I have submitted PR for crash fix #4986.
As I said earlier, Npp does not support wildcard for file opening, so this PR just fixes the crash and will not open anything.

**Comment by donho** [comment]
Cannot reproduce the crash at all.
